aa-rest/src/test/java/org/uniprot/api/aa/controller/ArbaGetByIdControllerIT.java

```diff
@@ -148,7 +148,7 @@ public GetIdParameter withFilterFieldsParameter() {
                     .id(ARBA_ID)
                     .fields("annotation_covered")
                     .resultMatcher(jsonPath("$.uniRuleId", is(ARBA_ID)))
-                    .resultMatcher(jsonPath("$.information").doesNotExist())
+                    .resultMatcher(jsonPath("$.information").exists())
```

Comment:
We should also check if information.oldRuleNum exists and preferably with actual value.